Formula/libgsm.rb

```diff
@@ -31,8 +31,10 @@ def install
     man1.mkpath
     man3.mkpath
 
+    inreplace "Makefile", "libgsm.1.0.13.dylib", "libgsm.1.0.16.dylib"
```

Comment:
We'll need this reported upstream and a new release tarball will likely be needed.